plutus-playground-server/test/Playground/UsecasesSpec.hs

```diff
@@ -175,15 +194,23 @@ vestingTest =
             , program =
                   toJSONString
                       [ CallEndpoint
-                            { endpointName = EndpointName "vest funds"
-                            , wallet = w2
-                            , arguments = toEndpointParam ()
+                            { caller = w2
+                            , argumentValues =
+                                  FunctionSchema
+                                      { endpointName = EndpointName "vest funds"
+                                      , arguments = toEndpointParam ()
+                                      }
                             }
                       , AddBlocks 40
                       , CallEndpoint
-                            { endpointName = EndpointName "retrieve funds"
-                            , wallet = w1
-                            , arguments = toEndpointParam $ Ada.adaValueOf 8
+                            { caller = w1
+                            , argumentValues =
+                                  FunctionSchema
+                                      { endpointName =
+                                            EndpointName "retrieve funds"
+                                      , arguments =
+                                            toEndpointParam $ Ada.adaValueOf 8
```

Comment:
does this need to be qualified? there's only one place `adaValueOf` and `lovelaceValueOf` are going to come from.